generated/artifacts/openapi-yaml/openapi/openapi.yaml

```diff
@@ -0,0 +1,5403 @@
+      type: object
+    us_driving_licence_breakdown_address:
+      description: Asserts whether the address data provided matches a real driving
+        license in the DMV driver's license database.
+      properties:
+        result:
+          type: string
+        breakdown:
+          $ref: '#/components/schemas/us_driving_licence_breakdown_address_breakdown'
+      type: object
+    us_driving_licence_breakdown_personal_breakdown:
+      properties:
+        first_name:
+          $ref: '#/components/schemas/document_breakdown_data_comparison_breakdown_issuing_country'
+        name_suffix:
+          $ref: '#/components/schemas/document_breakdown_data_comparison_breakdown_issuing_country'
+        height:
+          $ref: '#/components/schemas/document_breakdown_data_comparison_breakdown_issuing_country'
+        weight:
+          $ref: '#/components/schemas/document_breakdown_data_comparison_breakdown_issuing_country'
+        sex_code:
+          $ref: '#/components/schemas/document_breakdown_data_comparison_breakdown_issuing_country'
+        eye_color:
+          $ref: '#/components/schemas/document_breakdown_data_comparison_breakdown_issuing_country'
+        date_of_birth:
+          $ref: '#/components/schemas/document_breakdown_data_comparison_breakdown_issuing_country'
+        last_name:
+          $ref: '#/components/schemas/document_breakdown_data_comparison_breakdown_issuing_country'
+        middle_name:
+          $ref: '#/components/schemas/document_breakdown_data_comparison_breakdown_issuing_country'
+        first_name_fuzzy:
+          $ref: '#/components/schemas/document_breakdown_data_comparison_breakdown_issuing_country'
+        middle_name_fuzzy:
+          $ref: '#/components/schemas/document_breakdown_data_comparison_breakdown_issuing_country'
+        last_name_fuzzy:
+          $ref: '#/components/schemas/document_breakdown_data_comparison_breakdown_issuing_country'
+        middle_initial:
+          $ref: '#/components/schemas/document_breakdown_data_comparison_breakdown_issuing_country'
+      type: object
+    us_driving_licence_breakdown_personal:
+      description: Asserts whether the personal data provided matches a real driving
+        license in the DMV driver's license database.
+      properties:
+        result:
+          type: string
+        breakdown:
+          $ref: '#/components/schemas/us_driving_licence_breakdown_personal_breakdown'
+      type: object
+    device_intelligence_breakdown_breakdown_device_breakdown:
+      properties:
+        application_authenticity:
+          $ref: '#/components/schemas/document_breakdown_data_comparison_breakdown_issuing_country'
+        device_integrity:
+          $ref: '#/components/schemas/document_breakdown_data_comparison_breakdown_issuing_country'
+        device_reputation:
+          $ref: '#/components/schemas/document_breakdown_data_comparison_breakdown_issuing_country'
```

Comment:
where's this schema defined? can't find its file 🤔

Reply:
It's not a file, but a path relative the file itself:
```yaml
    document_breakdown_data_comparison_breakdown_issuing_country:
      properties:
        result:
          type: string
        properties:
          type: object
      type: object
```

As the fields are the same, generator seems to factorise with a single name (`document_breakdown_data_comparison_breakdown_issuing_country`.

Reply:
isn't this an alpha-3 country code then?

Reply:
I was surprised also and I'll dig further as soon as time will allow; anyway it was defined like that before this PR already:
https://github.com/onfido/onfido-openapi-spec/blob/e09e9db616b0abbb50c463a4c369420df2015630/schemas/reports/right_to_work_breakdown.yaml#L29-L35

Reply:
not arguing about that, but if it's an alpha-3 country code in practice, I think we should document towards it.